src/routes/Layout.jsx

```
@@ -5,10 +5,10 @@ import useIsMobile from '../hooks/header/useIsMobile'
 import Footer from './../components/web/Footer'
 import ScrollBtn from '../components/common/ScrollAndWriteButton'
 
-const Layout = ({ children, headerType, label, noMargin = false }) => {
+const Layout = ({ children, headerType, label, noMargin = false, paddingX = false }) => {
```

ESLint

[error] 'children' is missing in props validation

(react/prop-types)